vcd/resource_vcd_org.go

```diff
@@ -417,11 +417,11 @@ func resourceOrgUpdate(_ context.Context, d *schema.ResourceData, m interface{})
 	}
 
 	log.Printf("[TRACE] Org %s updated", orgName)
-	return nil
+	return resourceOrgRead(ctx, d, m)
```

Comment:
Note to reviewers: I noticed a Read was missing after Update. I assumed this is wrong, but let me know otherwise.